### DISCUSSION TITLE
I`m an Architect, Korean.

### DISCUSSION THREAD

**Discussion by SamSoJang** [cnr-isti-vclab/meshlab#1361] [Show and tell]
This is not a question.
I use meshlab to open 3d scanned objects.
I use iphone pro to get 3d information of a building.
Today I got a message that I opened 100 meshed so I`m using meshlab often.
The message writer think that I got an idea about this software.

I`m sorry that English is not my native language so these texts may be odds.

Frankly, I have no idea about this software. I don`t know how use this app. I just open files and measure distance between walls.

I wish I can move copy and erase 3d objects, but it is not possible.

This is not a question. This texts are a small experience.

And I want to say thank you for this software. This app is the only way to see my 3d files in my PC. iphone screen is too small.

Thank you.

**Comment by jmespadero** [comment]
Hi SamSoJang,

I have contributed only small parts to meshlab, but I would like to thank you the effort you have taken to let know that meshlab is being useful for you. Big respect to people being so wholesome. 

Regards.
